content/en/community/hackathons/sessions/baby-steps/content/intro.md

```diff
@@ -16,21 +16,19 @@ It is located in the [main Unikraft repository](https://github.com/unikraft/unik
 `Libraries` are additional software components that will be linked against Unikraft for the final image.
 There are multiple supported libraries.
 Each unikernel image is using its specific libraries.
-Libraries are also called **external** libraries as they sit outside the main Unikraft repository.
-Libraries are typically common libraries (such as OpenSSL or LwIP) that have been ported on top of Unikraft.
+Libraries are also called **external** libraries, as they sit outside the main Unikraft repository.
+They are typically common libraries (such as OpenSSL - a crypto library, or LwIP - a TCP/IP network stack) that have been ported on top of Unikraft.
 They are located in specialized repositories in the [Unikraft organization](https://github.com/unikraft/), those whose names start with `lib-`.
 
 `Application` is the actual application code.
 It typically provides the `main()` function (or equivalent) and is reliant on Unikraft and external libraries.
 Applications that have been ported on top of Unikraft are located in repositories in the [Unikraft organization](https://github.com/unikraft/), those whose names start with `app-`.
 
 An important role of the core Unikraft component is providing support for different platforms and architectures.
-A platform is the virtualization / runtime environment used to run the resulting image.
-An architecture details the CPU and memory specifics that will run the resulting image.
+A platform is the virtualization / runtime environment used to run the resulting image (i.e QEMU, Firecracker, VMWare etc.).
+An architecture details the CPU and memory specifics that will run the resulting image (i.e. x86, ARM).
 
-As this is a rather complicated setup, a companion tool ([kraft](https://github.com/unikraft/kraft)) was designed and implemented to provide the interface for configuring, building and running unikernel images based on Unikraft.
-The recommended way of building and running Unikraft is via `kraft`.
+We can either use a lower-level configuration and build system (based on [Kconfig](https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html) and [make](https://www.gnu.org/software/make/)) to get a grasp of how everything works, or use the [scripts](https://github.com/unikraft-upb/scripts) presented in the previous session for a more simplified approach in quickly building and running certain applications.
```

Comment:
There is another mention of the scripts here. @StefanJum not sure if we want to remove this one as well.